src/foam/nanos/crunch/CapabilityCategory.js

```diff
@@ -75,7 +75,6 @@ foam.CLASS({
       javaThrows: ['AuthorizationException'],
       javaCode: `
         if ( getVisibilityCondition() == null ) {
-          ( getDefaultAuthorizer()).authorizeOnRead(x, this);
           return;
```

Comment:
### Just a note on this:
Either we added **capabilitycategory.read.*** to each group that needed to see crunch categories - or we accept that everyone should see categories.

Latter makes more sense - discussed originally by @noodlemoodle - just appending this to this pr